openwrt-openwisp-monitoring/tests/env1.lua

```diff
@@ -0,0 +1,79 @@
+package.path = package.path .. ";../files/lib/?.lua"
+
+local env = {}
```

Comment:
This file contains all openwrt mocks that are used for testing functions. Will rename the file too